sys/linux/socket_alg.txt

```diff
@@ -19,6 +19,9 @@ accept$alg(fd sock_alg, peer const[0], peerlen const[0]) sock_algconn
 accept4$alg(fd sock_alg, peer const[0], peerlen const[0], flags flags[accept_flags]) sock_algconn
 sendmsg$alg(fd sock_algconn, msg ptr[in, msghdr_alg], f flags[send_flags])
 sendmmsg$alg(fd sock_algconn, mmsg ptr[in, array[msghdr_alg]], vlen len[mmsg], f flags[send_flags])
+recvmsg$alg(fd sock_algconn, msg ptr[inout, msghdr_alg], f flags[recv_flags])
+recvmmsg$alg(fd sock_algconn, mmsg ptr[inout, array[msghdr_alg]], vlen len[mmsg], f flags[recv_flags])
+read$alg(fd sock_algconn, buf buffer[out], size len[buf])
```

Comment:
Do we need all 3? Do they make difference for AF_ALG sockets? If not, let's do just `read`. recvmsg/mmsg are more complex to generate and syzkaller will call them on alg sockets with existing descriptions too.

Reply:
I see, thanks, I've removed recvmsg/mmsg. read() also leads to same entry points(skcipher_recvmsg etc).